aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -140,8 +140,8 @@ impl Aggregator {
     /// Returns true if addition succeeded, and false if it would overflow.
     pub fn try_add(
         &mut self,
-        resolver: &dyn AggregatorResolver,
```

Comment:
we had half of the calls with resolver first, and half with last. Made them consistent, with some noise on this PR

Reply:
Nice!